<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imgInfoDialog</class>
 <widget class="QDialog" name="imgInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>图像信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,1">
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>182</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>182</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>文件类型：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lab_imgKind">
           <property name="text">
            <string>imgKind</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>文件名称：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lab_imgName">
           <property name="text">
            <string>imgName</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>文件格式：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lab_imgFormat">
           <property name="text">
            <string>imgFormat</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>文件位深：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lab_imgDepth">
           <property name="text">
            <string>imgDepth</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>文件大小：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lab_imgData">
           <property name="text">
            <string>imgData</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>文件尺寸：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lab_imgSize">
           <property name="text">
            <string>imgSize</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>文件储存路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_imgAbPath">
       <property name="text">
        <string>imgAbPath</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>文件创建时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_imgCreateTime">
       <property name="text">
        <string>imgCreateTime</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>imgInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>imgInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
